src/commands/general/delete_interaction.rs

```diff
@@ -0,0 +1,36 @@
+use eyre::Result;
```

Comment:
(on mobile so no actual suggestion :trollface: ) as we're giving an explicit error variant to Result with `crate::Error` this isn't needed